UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -554,20 +529,13 @@ private void SetConsciousState(ConsciousState NewConsciousState)
 		{
 			if (damageSplit)
 			{
-				float bodyParts = RootBodyPartContainers.Sum(Container => Container.ContainsLimbs.Count());
+				float bodyParts = BodyPartList.Count;
 				damage /= bodyParts;
 			}
 
-			foreach (var Container in RootBodyPartContainers)
+			foreach (var bodyPart in BodyPartList)
 			{
-				if (damageSplit)
```

Comment:
what happened to damage Split?

Reply:
since there aren't body parts inside of body parts, the damagesplit dmg value is calculated a few lines above and then applied to all

Reply:
there could still be multiple in the same area so for example implanting multiple arms

Reply:
solved on last commit